Functions.Templates/Templates/DaprServiceInvocationTrigger-JavaScript/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "defaultFunctionName": "DaprServiceInvocationTrigger",
+  "description": "$DaprServiceInvocationTrigger_description",
```

Comment:
Please add a matching description string to the resource file located at /azure-functions-templates/Functions.Templates/Resources/Resources.resx